Assets/Scripts/Ditch.cs

```diff
@@ -0,0 +1,35 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEngine.SceneManagement;
+
+public class Ditch : MonoBehaviour {
+
+	private GameObject player;
+
+	// Use this for initialization
+	void Start () {
+	
+	}
+	
+	// Update is called once per frame
+	void Update () {
+	
+	}
+
+	void OnTriggerEnter2D(Collider2D other) {
+		if (other.gameObject.CompareTag ("Player")) {
+			player = other.gameObject;
+			StartCoroutine ("Fall");
+		}
+	}
+
+	IEnumerator Fall() {
```

Comment:
[JUST A COMMENT] Is this how you run while loops across multiple update frames? That's pretty cool!